Description: Correct "normalize" program and package name.
 "normalize" Debian package was renamed to "normalize-audio".
Author: Maxime Chatelle <xakz@rxsoft.eu>
Bug-Debian: http://bugs.debian.org/589790
Origin: vendor
Forwarded: not-needed
<<<<<<< HEAD
Last-Update: 2016-10-13
--- a/pacpl
+++ b/pacpl
@@ -2524,8 +2524,8 @@
=======
Last-Updated: 2011-06-20

Index: git/pacpl
===================================================================
--- git.orig/pacpl	2014-10-12 22:43:43.309656758 +0200
+++ git/pacpl	2014-10-12 22:43:43.309656758 +0200
@@ -2535,8 +2535,8 @@
>>>>>>> ppa/5.0.1
        }
        
        # normalize wav file before encoding to output format
-       perror("no_app","normalize") if $normalize and not `which normalize 2>/dev/null`;
-       system("normalize $nopts \"$outf.$$.wav\"") if $normalize;
+       perror("no_app","normalize") if $normalize and not `which normalize-audio 2>/dev/null`;
+       system("normalize-audio $nopts \"$outf.$$.wav\"") if $normalize;
 
        # copy meta-data from input file to %tag_name 
        read_tags("$inf.$infmt", "$if") if $run{$if}{TAGS}{READ} == 1 and not $dryrun;
